### PR TITLE
Add ActivityImageType property

### PR DESCRIPTION
**Pull request by BillZengMSFT**

### CODE REVIEW

File: Node/src/models/o365ConnectorCard.ts [hunk 1]
```diff
@@ -166,6 +166,18 @@ export class O365ConnectorCardSection implements teams.IIsO365ConnectorCardSecti
         return this;
     }
 
+    public activityImageType(imageType: string): this {
+        if (imageType) {
+            if (imageType === 'avatar' || imageType === 'article')
```
Comment (jingjingxia) [inline Node/src/models/o365ConnectorCard.ts]: Is this case sensitive or not? I think we should not require exact match.

Comment (jingjingxia) [reply Node/src/models/o365ConnectorCard.ts]: Can we consider introducing enum for these values?

Comment (aosolis) [inline Node/src/models/o365ConnectorCard.ts]: 'avatar' and 'article' should be defined as constants, perhaps like ActivityTypes in C#. Discoverability of the valid values for these strings is already a problem.

File: Node/src/models/o365ConnectorCard.ts [hunk 2]
```diff
@@ -166,6 +166,18 @@ export class O365ConnectorCardSection implements teams.IIsO365ConnectorCardSecti
...
+                this.data.activityImageType = imageType;
+            else
```
Comment (aosolis) [inline Node/src/models/o365ConnectorCard.ts]: braces around if/else blocks please

File: Node/src/models/index.d.ts [hunk 3]
```diff
@@ -184,6 +184,11 @@ export interface IO365ConnectorCard {
   potentialAction?: IO365ConnectorCardActionBase[];
 }
 
+export enum O365ConnectorCardActivityImageTypes {
```
Comment (aosolis) [inline Node/src/models/index.d.ts]: Missing doc comments.

File: Node/src/models/index.d.ts [hunk 4]
```diff
@@ -200,6 +205,8 @@ export interface IO365ConnectorCard {
  *
  * @member {string} [activityImage] Activity image
  *
+ * @member {string} [activityImageType] Activity image type
+ *
  * @member {boolean} [markdown] Use markdown for all text contents. Default vaule is true.
```
Comment (aosolis) [inline Node/src/models/index.d.ts]: Typo: "value".

File: Node/src/models/index.d.ts [hunk 5]
```diff
@@ -200,6 +205,8 @@ export interface IO365ConnectorCard {
  *
  * @member {string} [activityImage] Activity image
  *
+ * @member {string} [activityImageType] Activity image type
```
Comment (aosolis) [inline Node/src/models/index.d.ts]: Document the default value if not specified.

**Pull request merged**